README.rst

```diff
@@ -26,12 +26,12 @@ Features:
  * saving to PIL_ or Pillow_ image memory
  * some back-ends are based on this discussion: http://stackoverflow.com/questions/69645/take-a-screenshot-via-a-python-script-linux
  * pure Python library
- * supported Python versions: 2.7, 3.4, 3.5, 3.6, 3.7
+ * supported Python versions: 2.7 and 3.5+
```

Comment:
I prefer explicit versions: 3.5, 3.6, 3.7

Reply:
My thinking was that it's one less line to update when 3.8 is released, but would you like a new PR to add back explicit versions?

Reply:
I see you've taken care of it in https://github.com/ponty/pyscreenshot/commit/f78816044e5555b7b62abdf4450b022cc7cf8ed9. 👍